### ISSUE TITLE
if current parquet file is empty, it will stop iterate instead of iterate to next parquet file for reading data, not right

### ISSUE THREAD

**Issue by wwang-talend** [delta-io/connectors#125]
Please see:

https://github.com/delta-io/connectors/blob/master/standalone/src/main/scala/io/delta/standalone/internal/data/CloseableParquetDataIterator.scala#L85

**Comment by zsxwing** [comment]
Good catch! Thanks for reporting the issue. Feel free to submit a PR to fix it if you have time!

**Comment by wwang-talend** [comment]
thanks, i will privide the pr later.
